Description: [PATCH] Obtain gsettings transparency key value before showing panel
 #1124 When creating a panel, the transparency key value needs to be obtained
 before showing the panel. This allows transparency panel gsettings overrides.
Author: David Mohammed <fossfreedom@ubuntu.com>
Forwarded: https://github.com/budgie-desktop/budgie-desktop/pull/1251
Last-Update: 2018-08-07

---
 src/panel/manager.vala | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/src/panel/manager.vala b/src/panel/manager.vala
index de580923..11d501cb 100644
--- a/src/panel/manager.vala
+++ b/src/panel/manager.vala
@@ -1301,6 +1301,10 @@ public class PanelManager : DesktopManager
         load_panel(uuid, false);

         set_panels();
+
+        string path = this.create_panel_path(uuid);
+        var settings = new GLib.Settings.with_path(Budgie.TOPLEVEL_SCHEMA, path);
+        transparency = (PanelTransparency)settings.get_enum(Budgie.PANEL_KEY_TRANSPARENCY);
         show_panel(uuid, position, transparency);

         if (new_defaults == null || name == null) {
--
2.14.1

